js/mixins/responsiveManagerMixin.js

```diff
@@ -28,6 +29,10 @@ var ResponsiveManagerMixin = {
 
   generateResponsiveData: function() {
     var componentWidth = ReactDOM.findDOMNode(this).getBoundingClientRect().width;
```

Comment:
I know this is in the original code too, but if ReactDOM.findDOMNode(this) fails, we'll through an error.  Can you store it in a local variable and make sure it's valid?